File: Python/7py 2/meeting.py
"""
|--------------------------------------------------------------------------
| The Office IV - Find a Meeting Room
|--------------------------------------------------------------------------
|
"""

def meeting(rooms):
   return rooms.index('O') if 'O' in rooms else 'None available!'


def meeting(rooms):
   try:
      return rooms.index("O")
   except:
      return "None available!"


def meeting(rooms):
   return next((i for i, r in enumerate(rooms) if r == 'O'), 'None available!')


print(meeting(['X', 'O', 'X']), 1)
print(meeting(['O','X','X','X','X']), 0)
print(meeting(['X','X','O','X','X']), 2)
print(meeting(['X']), 'None available!')

File: JavaScript/7js 2/meeting.js
/*
|--------------------------------------------------------------------------
| The Office IV - Find a Meeting Room
|--------------------------------------------------------------------------
|
*/

function meeting(x) {
   return x.indexOf("O") < 0 ? 'None available!' : x.indexOf("O");
}


function meeting(x) {
   return !x.includes(`O`) ? `None available!` : x.indexOf(`O`);
}


function meeting(x) {
   return (a = x.indexOf('O')) > -1 ? a : 'None available!';
}


console.log(meeting(['X', 'O', 'X']), 1);
console.log(meeting(['O', 'X', 'X', 'X', 'X']), 0);
console.log(meeting(['X', 'X', 'X', 'X', 'X']), 'None available!');